runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaClientConnectionPool.java

```diff
@@ -1280,6 +1280,7 @@ private void doConnectionBegin(
                             .destination(host)
                             .sourcePort(0)
                             .destinationPort(port)))
+                        .infos(i -> i.item(ii -> ii.authority(host)))
```

Comment:
This should be coming from `ProxyBeginEx` `info` `authority`, not `host`, and `address` key for shared connections should include `authority` in addition to `host` and `port`, agree?

Otherwise, two streams differing only by `authority` would be treated as equal, even though their TLS handshakes would have a different `sni`.